### PR TITLE
Add Apollo 11 picture of Earth from NASA

### PR DESCRIPTION
**Pull request by win8linux**
## Description

Picture of the Earth from the Apollo 11 mission of NASA

### Image Information

- Author: NASA
- License: Public Domain
- Source: [Webpage](https://images.nasa.gov/details-as11-44-6551), [Original image](https://images-assets.nasa.gov/image/as11-44-6551/as11-44-6551~orig.jpg)

Source image has been cropped to 4,152 × 2,336 size.

### Submitter Checklist

- [x] Submitter has reviewed the Image Requirements
- [x] Squashed commits with `git rebase -i` (if needed)
- [x] Updated the respective `meson.build` and `xml.in` files

### CODE REVIEW

File: backgrounds/meson.build [hunk 1]
```diff
@@ -1,6 +1,7 @@
 backgrounds = [
     'temp.jpg',
-    'temp2.png',
+    'temp2.png'
```
Comment (serebit) [inline backgrounds/meson.build:3]: Removing the comma here breaks the build.

File: data/budgie-backgrounds.xml.in [hunk 2]
```diff
@@ -17,4 +17,12 @@
     <scolor>#000000</scolor>
     <shade_type>solid</shade_type>
   </wallpaper>
+  <wallpaper deleted="false">
+    <name>apollo-11-earth.jpg</name>
```
Comment (serebit) [inline data/budgie-backgrounds.xml.in:21]: Should change to "Earth from Apollo 11".

**Review by serebit — CHANGES_REQUESTED** [review]
Great pic that I would support adding, but there are some things that need to be addressed first.

**Review by JoshStrobl — CHANGES_REQUESTED** [review]
Needs fixes from @serebit's review otherwise LGTM

**Pull request merged**